### PR TITLE
[1561247] Add kubeconfig to openshift_bootstrap_autoapprover

### PR DESCRIPTION
**Pull request by mtnbikenc**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1561247

### CODE REVIEW

**Comment by mtnbikenc** [comment]
@michaelgugino 
> Do we want to put the --config line in both commands?

I was considering that but in looking at the many other places we use `oc` there is not a consistent pattern of using it everywhere.  Since this bug didn't mention a failure at that task, I left it alone. (except for switching to using `command`)

File: roles/openshift_bootstrap_autoapprover/tasks/main.yml [hunk 1]
```diff
@@ -20,13 +20,14 @@
 
 # TODO: temporary until we fix apply for image stream tags
 - name: Remove the image stream tag
-  shell: >
+  command: >
     {{ openshift_client_binary }} delete -n openshift-infra istag node:v3.10 --ignore-not-found
```
Comment (smarterclayton) [inline roles/openshift_bootstrap_autoapprover/tasks/main.yml]: Add it here too

**Comment by mtnbikenc** [comment]
@smarterclayton Updated

**Pull request merged**